src/app/src/components/QuizHint.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Box } from 'rebass';
+import { Text } from './custom-styled-components';
+import { themeGet } from 'styled-system';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+const StyledQuizHint = styled(Box)`
+    background: ${themeGet('colors.greens.3')};
+    text-align: center;
+    width: fit-content;
+    margin-left: auto;
+`;
+
+const QuizHint = ({ hint }) => {
+    return (
```

Comment:
Is it ever possible for `hint` to be undefined here? If not, consider adding an `.isRequired` proptype for it and dropping the `{ return` in favor of `({ hint }) => (<StyledQuizHint ....`.

If so, consider adding either a default prop for hint or handling it inline here.

If so, consider either

Reply:
Good idea! There will always be a hint.